src/runtime/HalideRuntime.h

```diff
@@ -294,7 +294,15 @@ extern void halide_memoization_cache_cleanup();
  * The raw representation of an image passed around by generated
  * Halide code. It includes some stuff to track whether the image is
  * not actually in main memory, but instead on a device (like a
- * GPU). */
+ * GPU).
```

Comment:
This comment was cut intentionally because it didn't seem like it belonged in a user-facing file.